storage/bucket.go

```diff
@@ -337,6 +337,9 @@ type BucketAttrs struct {
 	// Typical values are "multi-region", "region" and "dual-region".
 	// This field is read-only.
 	LocationType string
+
+	// The project number of the project the bucket belongs to.
```

Comment:
Note that the field is read-only (as in L339 above).

Reply:
Done. Added the comment.